### PR TITLE
Added permission of execution in the bin/add-roles.sh, bin/local-run-ga.sh and bin/process-and-import-general-apps-ccd-definition.sh scripts

### PR DESCRIPTION
**Pull request by LeonardoPalmeiro**
**Before creating a pull request make sure that:**

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###
To finally resolve **_Permission denied_** in bin/add-roles.sh, bin/local-run-ga.sh and bin/process-and-import-general-apps-ccd-definition.sh
Basically I added execution permission with chmod +x.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by TatsyMaslova** [comment]
I cant see your changes Leo. the files are empty

**Review by Gareth40343 — APPROVED** [review]

**Comment by laarmada** [comment]
we dont use in CUI that [bin/process-and-import-general-apps-ccd-definition.sh].

we use: /bin/process-and-import-ccd-definition.sh
